### PR TITLE
feat(iam): create & edit resource groups

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `rework/iam-rollback-architecture`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-10401, #MANAGER-10518
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.styles.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.iam-create_resource_group {
+  max-width: 40rem;
```
Comment (anooparveti) [inline packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.styles.scss:2]: **nitpick:** Use bootstrap utility instead.

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.controller.js [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  get mode() {
+    return this.resourceGroup ? 'edit' : 'create';
+  }
+
+  /**
+   * A contextualized Set where each key map to a translation
+   * @returns {Object<string,string>}
+   */
+  get translations() {
+    const pfx = 'iam_create_resource_group';
```
Comment (anooparveti) [inline packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.controller.js:119]:
```suggestion
    const prefix = 'iam_create_resource_group';
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.template.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                class="oui-button oui-button_l oui-button_link mr-3"
+                data-ng-disabled="$ctrl.isSubmitting"
+                data-ng-click="$ctrl.cancelCreation()"
+            >
+                <span data-translate="iam_create_resource_group_cancel"></span>
+            </button>
+            <button
+                type="submit"
+                class="oui-button oui-button_l oui-button_primary mr-3"
+                data-ng-disabled="!$ctrl.canSubmit"
```
[the remark this answers is not in the document]
Comment (y4nnL) [reply packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.template.html:122]: The `canSubmit` getter handles this
```javascript
  get canSubmit() {
    return (
      !this.isSubmitting && this.form?.$valid && !this.resourceGroup?.readOnly
    );
  }
```

File: packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.styles.scss (hunk 1 above)
Comment (y4nnL) [reply packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.styles.scss:2]: Can you tell me witch one ?

Comment (anooparveti) [reply packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.styles.scss:2]: Okay, I see we have only `mw-100` with respect to `max-width`. 
But can you use row and have a col of fixed width always?

**Review by anooparveti — APPROVED** [review]

**Pull request merged**